src/components/appointments/DateSelect.stories.mdx

```diff
@@ -0,0 +1,80 @@
+import {Canvas, Meta, Story} from '@storybook/addon-docs';
+import {expect} from '@storybook/jest';
+import {userEvent, within} from '@storybook/testing-library';
+
+import {ConfigDecorator, FormikDecorator} from 'story-utils/decorators';
+
+import DateSelect from './DateSelect';
+import {mockAppointmentDatesGet} from './mocks';
+
+export const Template = () => <DateSelect />;
+
+<Meta
+  title="Private API / Appointments / DateSelect"
+  decorators={[FormikDecorator, ConfigDecorator]}
+  component={DateSelect}
+  parameters={{
+    formik: {
+      initialValues: {
+        products: [{product: 'e8e045ab', amount: 1}],
+        location: '1396f17c',
+        date: '',
+      },
+    },
+    msw: {
+      handlers: [mockAppointmentDatesGet],
+    },
+  }}
+/>
+
+## Disable dates outside of range
+
+<Canvas>
+  <Story name="Calendar with available date range">{Template.bind({})}</Story>
+</Canvas>
+
+## Disable missing dates within range
+
+<Canvas>
+  <Story
+    name="Calendar with disabled gaps in date range"
```

Comment:
I think the difference between these two stories is not clear without looking at the code?
![Screenshot from 2023-06-13 12-34-42](https://github.com/open-formulieren/open-forms-sdk/assets/19154114/42b2975a-e75f-4650-87da-796ed8a714d1)